cmd/create/machinepool/nodepool.go

```diff
@@ -387,51 +387,54 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		npBuilder.TuningConfigs(inputTuningConfig...)
 	}
 
-	var inputKubeletConfigs []string
 	kubeletConfigs := args.kubeletConfigs
-	// Get the list of available kubelet configs
-	availableKubeletConfigs, err := r.OCMClient.ListKubeletConfigNames(cluster.ID())
-	if err != nil {
-		r.Reporter.Errorf("%s", err)
-		os.Exit(1)
-	}
-	if kubeletConfigs != "" {
+
```

Comment:
I have followed the existing patterns and not done a large refactor on this PR to support unit testing as we have that work in progress under another epic.

I have manually verified the changes work as expected. See the description of the PR for details.